frontend/www/js/omegaup/components/common/Confirmation.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="confirmation">
+    <form class="panel panel-default popup">
+      <button class="close" type="button" v-on:click="$emit('close')">×</button>
```

Comment:
```suggestion
      <button class="close" type="button" v-on:click.prevent="$emit('close')">×</button>
```

así se evita que la forma se envíe.